.travis.yml

```diff
@@ -18,6 +18,8 @@ before_script:
   - kind load docker-image kubernetes/pause
   - cp "$(kind get kubeconfig-path --name="kind")" /tmp/admin.conf
 script:
+- mkdir -p ~/gopath/src/sigs.k8s.io/
+- mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
```

Comment:
is the dot at the end intentional? just asking does not matter much.

Reply:
Just my convention. I can remove it if needed.

Reply:
i am fine, have not seen that frequently (for current dir at the end of the path)  so asking.